### PR TITLE
Re-enable AutoML model create tests

### LINKED ISSUE #3706: Re-enable model test
`python-docs-samples/vision/automl/model_test.py`

## Describe the issue

After  [Pull Request 3693](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/3693), there is only one test in this file: `test_model_create_status_delete`, and it is currently skipped. 

This should be re-enabled.

### PR DESCRIPTION
**Pull request by busunkim96**
Closes #3706.

I don't have the context around why these tests were disabled. Does these tests need to be modified in some other way? 

https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1608 disabled these tests.

### CODE REVIEW

**Comment by tmatsuo** [comment]
Few things.

1. The tests are using the following code:

```
model_name = "test_" + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
```
This easily conflicts among multiple builds. Can you update this to use uuid?

2. The tests are calceling the operation, but it's a best effort trial of cancellation, so sometimes the models are created before the cancellation happens. I think we should try to delete the models in teardown code.

File: translate/automl/model_test.py [hunk 1]
```diff
@@ -26,29 +26,32 @@
...
-    # create model
+@pytest.fixture
+def model():
     client = automl.AutoMlClient()
-    model_name = "test_" + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+    model_name = "test_{}".format(uuid.uuid4()).replace("-", "")[:32]
     project_location = client.location_path(project_id, compute_region)
     my_model = {
         "display_name": model_name,
         "dataset_id": "3876092572857648864",
```
Comment (busunkim96) [inline translate/automl/model_test.py:36]: It looks like this dataset doesn't exist. Is there some standard AutoML dataset I can use? @sirtorry @nnegrey

**Comment by sirtorry** [comment]
these samples are outdated. samples and tests can be deleted. translate [doc](https://cloud.google.com/translate/automl/docs/models#automl_translate_create_model-python) and vision [doc](https://cloud.google.com/vision/automl/docs/train) reference other snippets.

**Comment by tmatsuo** [comment]
@sirtorry 
Thanks for the input.

I did a quick search for the region tag, and there are still used in multiple location. Although they're all translated page, can we really delete them now?

**Comment by sirtorry** [comment]
I'm going to submit a request for these samples to be replaced on our localised/non-english documentation. We shouldn't remove the samples until that's been done. In this case, should we re-enable these tests?

**Comment by busunkim96** [comment]
Opened https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3766 to track removing those samples.

I'm alright with closing this and leaving the tests disabled. @tmatsuo Is that alright with you?

**Comment by tmatsuo** [comment]
@busunkim96 Yes, it's alright!

**Pull request closed without merging**